classes/class-p4-settings.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+if ( ! class_exists( 'P4_Settings' ) ) {
+	/**
+	 * Class P4_Settings
+	 */
+	class P4_Settings {
+
+		/**
+		 * Option key, and option page slug
+		 * @var string
+		 */
+		private $key = 'planet4_options';
+
+		/**
+		 * Array of metaboxes/fields
+		 * @var array
+		 */
+		protected $option_metabox = array();
+
+		/**
+		 * Options Page title
+		 * @var string
+		 */
+		protected $title = '';
+
+		/**
+		 * Options Page hook
+		 * @var string
+		 */
+		protected $options_page = '';
+
+		/**
+		 * Constructor
+		 * @since 0.1.0
```

Comment:
What are all those since 0.1.0 lines in the comments?

Reply:
Yes we do not need versions - i will update with the proper docblock